### ISSUE TITLE
Oxlint parity is 99.0%, not 100% — 112 real divergences, 85 in import-next

### ISSUE THREAD

**Issue by ofri-peretz** [ofri-peretz/eslint#912]
The oxlint parity harness was fixed in #906 — it had been linting zero files and scoring that as 0% parity. With it actually running, the real number is **99.0%**, not 100%:

```
Running ESLint...        10944 finding(s)
Running oxlint+shims...  10832 finding(s)
shared:        10806
eslint-only:   112 (0 allowlisted)
oxlint-only:   0
parity rate:   99.0%
```

Measured locally on `main` (`385262fa6`) against the full harvested corpus, 3,971 fixtures across 22 plugins.

`oxlint-only` is **0**, so oxlint never over-reports — every divergence is a rule oxlint misses that ESLint catches. The check exits 1 on these, so the scheduled parity run stays red until they are explained or fixed.

## The 112, by rule

| count | rule |
|---:|---|
| 17 | `import-next/no-unused-modules` |
| 15 | `import-next/unambiguous` |
| 12 | `import-next/prefer-default-export` |
| 12 | `import-next/no-named-export` |
| 8 | `import-next/newline-after-import` |
| 7 | `import-next/no-extraneous-dependencies` |
| 5 | `nestjs-security/no-permissive-cors` |
| 3 | `import-next/no-nodejs-modules` |
| 3 | `import-next/no-commonjs` |
| 3 | `modernization/prefer-template-literal` |
| 2 | `browser-security/require-postmessage-origin-check` |
| 2 | `secure-coding/no-insecure-comparison` |
| 2 | `node-security/no-timing-unsafe-compare` |
| 2 | `secure-coding/detect-non-literal-regexp` |
| 2 | `secure-coding/no-unsafe-regex-construction` |
| 2 | `secure-coding/no-improper-sanitization` |
| 1 | `import-next/consistent-type-specifier-style` |
| 1 | `import-next/no-internal-modules` |
| 1 | `import-next/no-namespace` |
| 1 | `secure-coding/no-missing-authentication` |
| 1 | `modularity/enforce-rest-conventions` |
| 1 | `node-security/no-ssrf` |
| 1 | `import-next/prefer-node-protocol` |
| 1 | `import-next/no-import-module-exports` |
| 1 | `node-security/no-unsafe-buffer-alloc` |
| 1 | `secure-coding/no-sensitive-data-exposure` |
| 1 | `pg/no-floating-query` |
| 1 | `pg/no-unsafe-query` |
| 1 | `secure-coding/no-directive-injection` |
| 1 | `secure-coding/no-template-injection` |
| 1 | `secure-coding/no-unsafe-deserialization` |

## Shape of it

**82 of 112 are `import-next/*`** across 12 rules — that concentration suggests one cause in the import-next shim rather than 12 unrelated gaps, and is where I would start. The remaining ~27 are spread thinly over security and quality rules, 1-5 each.

## What must not happen

These must not be allowlisted wholesale to turn the check green. The allowlist is for divergences with a recorded reason; 112 entries added to silence a gate is the same fabricated-measurement shape that #906 and #898 were about. Each entry needs either a fix or a stated reason oxlint cannot match ESLint there.

## Reproduce

```bash
npm run ilb:oxlint-parity:full
```

The per-finding list with file, rule, line and column is in `eslintOnlyUnexplained` in `benchmarks/results/ilb-oxlint-parity/<date>.json`.

**Comment by ofri-peretz** [comment]
Narrowing this before anyone spends time on the wrong hypothesis.

**It is not a config gap.** My first read was that the diverging rules were never enabled for oxlint — that was wrong, caused by looking for `import-next/*` keys. The parity config keys them by directory:

```
categories: all off  (correctness, suspicious, perf, pedantic, style, restriction, nursery)
rules: 375 total, of which interlace-import-next/* = 56
jsPlugins: interlace-import-next.cjs loaded
```

So both engines are asked the same questions, and every category is off so nothing extra leaks in. The 112 are **genuine behavioural differences between each ESLint rule and its oxlint shim**, which is the harder and more useful conclusion.

## Shape of the 85 import-next cases

They do not share one cause. They split by what the rule needs:

| rule | count | needs |
|---|---:|---|
| `no-unused-modules` | 17 | whole-program view — is this export used *anywhere* |
| `unambiguous` | 15 | per-file — does this file have module syntax |
| `prefer-default-export` | 12 | per-file |
| `no-named-export` | 12 | per-file |
| `newline-after-import` | 8 | per-file |
| `no-extraneous-dependencies` | 7 | package.json resolution |
| the rest | 14 | mixed, 1–3 each |

Only `no-unused-modules` is inherently multi-module — the accepted category that is *compatible but not efficient* under the oxlint JS plugin. The other ~68 are per-file or resolution-dependent, so they should be reachable.

## Suggested order

1. **The per-file cluster** (`unambiguous`, `prefer-default-export`, `no-named-export`, `newline-after-import` — 47 findings) has no structural reason to diverge. Most likely one shared assumption in the shim about what counts as a module. Highest return.
2. **`no-extraneous-dependencies`** (7) — resolution rooted differently under oxlint; check what package.json each side sees.
3. **`no-unused-modules`** (17) — allowlist with the multi-module reason rather than chase, consistent with the accepted position on `no-cycle`.

That last one is the only group I would allowlist, and only with the reason recorded. The other ~95 should be fixed or explained individually — a blanket entry to turn the gate green is the failure mode this whole thread has been removing.
